deployment/README.md

```diff
@@ -137,31 +103,53 @@ aws ssm put-parameter \
     --value "https://mmt.${MMT_STACK_STAGE}.maap-project.org"
 ```
 
-4. Generate CloudFormation template
+### 3. Set deployment configuration
+
+Deployment can be customized using any of the settings defined in `app/config.py`.
+
+To set values for these settings, create an `.env` file and add a line for the value of
+each setting (with the setting name prefixed by `MMT_STACK_`):
+
+```env
+MMT_STACK_certificate_arn="arn:aws:acm:us-west-2:12345:certificate/abc123"
+```
+
+#### Optional MCP config
+
+For deploying to MCP environments managed by NASA, you will need to specify values in `.env` for the `permissions_boundary_name` and `vpc_id` that should be used:
+
+```env
+MMT_STACK_permissions_boundary_name="example-permission-boundary"
+MMT_STACK_vpc_id="vpc-12345"
+```
+
+### 4. Generate CloudFormation template
 
 This step isn't required, but can be useful to just validate that the configuration.
 
 Synthesize the template to validate it basically works.
 
 ```bash
-$ npm run cdk synth
+export CDK_DEPLOY_ACCOUNT=$(aws sts get-caller-identity | jq .Account -r)
+export CDK_DEPLOY_REGION=$(aws configure get region)
+npm run cdk synth
 ```
 
-5. Deploy the Inception Pipeline
+### 5. Deploy the application
 
-This deploy step will deploy a CloudFormation Stack with a CodePipeline build that will then deploy the application stack.
+This deploy step will deploy a CloudFormation Stack for the MMT application.
 
 ```bash
 export CDK_DEPLOY_ACCOUNT=$(aws sts get-caller-identity | jq .Account -r)
-export CDK_DEPLOY_REGION=us-west-2
+export CDK_DEPLOY_REGION=$(aws configure get region)
 export MMT_STACK_STAGE="dit"
 
 $ npm run cdk deploy -- --require-approval never
 ```
 
-This pipeline will then deploy the application stack that creates a Postgres database, generates a docker image for the application, configures an ECS Task Definition and Service that uses that Task Definition, configures an application load balancer (ALB) to point to the ECS Service, and configures a custom DNS entry for the service.
+The application stack creates a Postgres database, generates a docker image for the application, configures an ECS Task Definition and Service that uses that Task Definition, configures an application load balancer (ALB) to point to the ECS Service, and configures a custom DNS entry for the service.
 
-6. Undeploy (optional)
+### Undeploy (optional)
```

Comment:
Undeploy is an interesting term - should we say "tear down" ?

Reply:
0b47c104c7e5a3a79bc71acbf3c006d0ff897155